test/test_app.py

```diff
@@ -0,0 +1,35 @@
+from unittest import mock
+from unittest.mock import MagicMock
```

Comment:
For Python 2.6, 2.7 builds to pass, we have to add a little bit of trickery:

```
try:
    import unittest2 as unittest
except ImportError:
    import unittest
```

and for mock, it looks like we can borrow some advice from [this guy](http://railslide.io/travis-mock-afternoon-of-frustration.html):
```
try:
    from mock import patch, mock_open
except ImportError:
    from unittest.mock import patch, mock_open
```

Can you please make these two changes? I am hoping the build will pass once this is done!